ui/src/chat/ChatMessage/ChatMessage.tsx

```diff
@@ -153,8 +153,14 @@ const ChatMessage = React.memo<
         f.uniq,
         f.take(3)
       )(seal.replied);
-      const lastReply = seal.replied.length ? _.last(seal.replied)! : '';
-      const lastReplyWrit = useWrit(whom, lastReply)!;
+      const repliesSortedByTime = seal.replied.sort((a, b) => {
+        const aTime = udToDec(a.split('/')[1]);
```

Comment:
probably should use `useChatState.getState().getTime(a)` and then use big int compare so we at least try to use server time if we have it.

Reply:
ok, just switched to times from getTIme().